src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -9,28 +9,22 @@
 using DelegateCommand = Dynamo.UI.Commands.DelegateCommand;
 using Dynamo.Models;
 using System.Windows.Data;
+using System.Globalization;
 
 namespace Dynamo.ViewModels
 {
-    public sealed class PathEnabledConverter : IValueConverter
+    public sealed class PathEnabledConverter : IMultiValueConverter
     {
-        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (value != null && parameter != null)
-            {
-                var disableStandardLibrary = (bool)parameter;
-                if (disableStandardLibrary)
-                {
-                    var path = value as string;
-                    return String.CompareOrdinal(path, Resources.PackagePathViewModel_Standard_Library) != 0;
-                }
-            }
-            return true;
+            var vm = values[0] as PackagePathViewModel;
+            var path = values[1] as string;
```

Comment:
Is there a chance of `values` being null here or chance for an out of range exception?

Reply:
since this is only used for the binding I don't think thats possible, but I can make it more robust just incase.